docs/tutorials/using-prisma.md

```diff
@@ -0,0 +1,152 @@
+---
+sidebar_position: 12
+title: Using Prisma
+description: Learn how to use Prisma with DBOS
+---
+
+[Prisma](https://www.prisma.io/) is a popular open-source TypeScript ORM.
+The main idea is to define your application data models (entities) in the [Prisma Schema](https://www.prisma.io/docs/orm/prisma-schema/overview), and then use Prisma to automatically generate [schema migrations](https://www.prisma.io/docs/orm/prisma-migrate/getting-started) as well as the corresponding [Prisma Client](https://www.prisma.io/docs/orm/prisma-client/setup-and-configuration/introduction) for you to manage your data.
+DBOS supports Prisma as an alternative to [Knex](https://knexjs.org/) and [TypeORM](https://typeorm.io) for transactional data management.
+
+### Getting Started
+
+An easy way to get started with Prisma is to bootstrap your application with our Prisma template.
+This is similar to the template used in the [quickstart](../getting-started/quickstart.md), but built with Prisma instead of Knex.
+To download it, run:
+
+```bash
+npx -y @dbos-inc/create@latest -t hello-prisma -n <app-name>
+```
+
+Then, build it, run schema migrations, and start the Prisma sample app:
+
+```bash
+npm run build
+npx dbos migrate
+npx dbos start
+```
+
+To see that it's working, visit this URL in your browser: [http://localhost:3000/greeting/dbos](http://localhost:3000/greeting/dbos).  You should get this message: `Greeting 1: Hello, dbos!` Each time you refresh the page, the counter should go up by one.
+
+### Setting Up Prisma Schema
+
+In DBOS, the Prisma Schema is defined in [the same way as any other Prisma project](https://www.prisma.io/docs/orm/prisma-schema/overview).
+In this tutorial we assume it is located at the default path (`prisma/schema.prisma`), for example:
+
+```javascript title="prisma/schema.prisma"
```

Comment:
Do we not support ```prisma ?

Reply:
No syntax highlighting for Prisma so far (we use Docusaurus built-in list of supported languages)